modyn/const/types.py

```diff
@@ -12,3 +12,24 @@ class TimeResolution(str, Enum):
     HOUR = "hour"
     MINUTE = "minute"
     SECOND = "second"
+
+
+TriggerEvaluationMode = Literal["hindsight", "lookahead"]
+"""
+- hindsight: The trigger is evaluated on the data that has already been seen. Once a we are certain about a threshold
+    crossing, the trigger is executed. Triggers are only performed after the threshold was crossed.
```

Comment:
This is only for performance triggers, though? it reads a bit general

Reply:
probably also for the cost triggers